Trappist/src/Promact.Trappist.Core/Controllers/CategoryController.cs

```diff
@@ -51,13 +52,13 @@ public IActionResult CatagoryAdd([FromBody] Category category)
         /// <param name="catagory">Object of  class Category</param>
         /// <returns>object of the class if key found or else it will return Bad request</returns>
         [HttpPut("{id}")]
-        public IActionResult CategoryEdit(int Id, [FromBody] Category category)
+        public async Task<IActionResult> CategoryEdit(int Id, [FromBody] Category category)
```

Comment:
give proper space. one space after region declaration and one space after every method completion.